sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/maskExpressions.scala

```diff
@@ -81,34 +83,38 @@ case class Mask(
     otherChar: Expression)
     extends QuinaryExpression
     with ImplicitCastInputTypes
```

Comment:
Do we still need this `ImplicitCastInputTypes` trait?

Reply:
yes, still overriding `def inputTypes: Seq[AbstractDataType]` from ImplicitCastInputTypes <- ExpectsInputTypes

Reply:
I think we can expect all the inputs to be string-type. `ExpectsInputTypes` should be enough here.

Reply:
Done, changed to ExpectsInputTypes